api/src/export/export_reference_patches.py

```
@@ -313,6 +314,267 @@ def fetch_geometries_by_label(token: str, label_id: int, table_name: str, bbox:
+					'patch_index': patch['patch_index'],
+					'resolution_cm': patch['resolution_cm'],
+					'label_id': label_id,
+					'layer_name': layer_name,
+					'validated': 1,
+					'area_m2': row.get('area_m2'),
+					'properties_json': json.dumps(row.get('properties') or {}, sort_keys=True),
+					'geometry': normalize_polygon_geometry(row['geometry']),
+				}
+			)
+		except Exception:
+			continue
```

Ruff (0.15.9)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)